docs/multilingual_multitask_evaluation.md

```diff
@@ -0,0 +1,255 @@
+# Multilingual Multitask Evaluation
```

Comment:
"Multilingual/Multitask" might be better to indicate that it's "and/or" not "and".

Similarly below, "multilingual and/or multitask evaluation" might be good.